### ISSUE TITLE
Display KML in the embedded map in our app

### ISSUE THREAD

**Issue by xordspar0** [xordspar0/Data61-capstone-2016#22]
Now that we won't be implementing our interface directly in NationalMap, we need our visualizations and styles to show up in our app. We will still have an export button that will allow the user to download a KML file that works with NationalMap and other map services.

**Comment by xordspar0** [comment]
The Google API for importing KML into a map only supports actual files that are accessible from the Web. Our code currently just makes the KML file in memory in the user's browser. We might have to write some server side code that could generate the KML file and serve it to Google.

**Comment by xordspar0** [comment]
I wrote a server-side Python script that posts the KML file to [pastebin.com](pastebin.com) as a temporary solution.

**Comment by xordspar0** [comment]
The pastebin API is pretty limited. Only 20 posts are allowed per day, so testing with this will be difficult, but it should work for the demo.

**Comment by xordspar0** [comment]
I've switched to another pastebin service, [paste.kde.org](paste.kde.org). Also, I discovered after many hours of troubleshooting that the Google API requires KML files to end in the ".kml" extension, otherwise the map is blank and there are no error messages saying what happened.

The routes show up properly on the map now, but it seems like embedded Google Maps have limited KML support; the styles are mostly ignored. At least there is some feedback to the user about the routes now. Unfortunately, the user still has to download the KML file to see the visualizations properly.
